var/spack/repos/builtin/packages/vtk/package.py

```diff
@@ -48,6 +48,7 @@ class Vtk(CMakePackage):
     variant('osmesa', default=False, description='Enable OSMesa support')
     variant('python', default=False, description='Enable Python support')
     variant('qt', default=False, description='Build with support for Qt')
+    variant('mpi', default=False, description='Enable MPI support')
```

Comment:
`mpi` variants should be default-`True`: https://spack.readthedocs.io/en/latest/packaging_guide.html#variant-names